docs/standards/smart-contracts/lsp0-erc725-account.md

```diff
@@ -96,13 +99,29 @@ Initiate an ownership transfer by setting the `newOwner` as `pendingOwner`.
 ### claimOwnership
 
 ```solidity
-function claimOwnership() public {
+function claimOwnership() public
 ```
 
 Transfers ownership of the contract to the `pendingOwner` address. Can only be called by the `pendingOwner`.
 
 _Triggers the **[OwnershipTransferred](#ownershiptransferred)** event once the new owner has claimed ownership._
 
+### renounceOwnership
+
+```solidity
+function renounceOwnership() public
+```
+
+Renounces ownership of the contract in 2 steps. Can only be called by the owner.
+
+First phase of this process will save the current `block.number` in `_lastBlock`.
+
+_Triggers the **[RenounceOwnershipInitiated](#renounceownershipinitiated)** event once the first phase of `renounceOwnership()` is complete._
+
+Second phase of this process will renounce ownership of the contract only if the current `block. number` is bigger than `_lastBlock + 100` and smaller than `_lastBlock + 200` 
+
+_Triggers the **[OwnershipTransferred](#ownershiptransferred)** event once the econd step of `renouneOwnership()` is complete._
```

Comment:
Maybe here we could try to replace `_lastBlock + 100` and `_lastBlock + 200` by some plain words, that explain more what this +100 and +200 mean?

Reply:
agree lets not mention `lastBlock ` as this is an implementation detail